zerver/actions/message_edit.py

```diff
@@ -977,7 +987,9 @@ def check_update_message(
         links_for_embed |= rendering_result.links_for_preview
 
         if message.is_stream_message() and rendering_result.mentions_wildcard:
-            stream = access_stream_by_id(user_profile, message.recipient.type_id)[0]
+            error = _("Invalid stream")
```

Comment:
The error message here was `Invalid stream ID` but I guess that does not make sense since user has not passed stream ID to the API. Though I am not sure what is the best error message for this case, but skipping ID still seems like improvement.